docs/nima/pico/pico_intro.adoc

```diff
@@ -36,11 +36,22 @@
 
 == Overview
 
-Helidon Pico //explain what it is and what it can do. 
+Helidon Pico is an optional feature in Helidon that provides service registry, a lifecycle engine and extensibility for customized code generation. Foundationally, Pico provides a way to develop declarative code using standard Javax/Jakarta annotation types (not Helidon-specific). The Helidon Pico framework provides a mix of declarative and programmatic ways to build your application. 
+
+Pico's minimalist, compile-time generated dependency injection (DI) framework and compile-time source code generation has a number of advantages, including:
+
+* pre-runtime validation of the dependency injection model
```

Comment:
First bullet should be:
* It enables declarative, Inversion of Control style of programming.